packages/cli/src/commands/composite/create.ts

```diff
@@ -23,12 +23,21 @@ export default class CreateComposite extends Command<Flags, { schemaFilePath: st
       char: 'o',
       description: 'path to the file where the composite representation should be saved',
     }),
+    index: Flags.boolean({
+      char: 'i',
+      description: 'If indexing should occur when creating composites (default=true)',
```

Comment:
Some command line flag parsers are bad at letting boolean flags be set to false.  So let's definitely double-check if we haven't already that the one we're using supports setting `--flagName=false`

Reply:
So the flags come from oclif, and there are no args to the flag. It will be either `--flagName` or `--no-flagName`

```// flag with no value (-f, --force)
  force: Flags.boolean({
    char: 'f',                    // short character for flag
    default: true,                // default value if flag not passed (can be a function that returns a boolean)
    env: 'MY_NAME',               // default value to the value of an environment variable
    // boolean flags may be reversed with `--no-` (in this case: `--no-force`).
    // The flag will be set to false if reversed. This functionality
    // is disabled by default, to enable it:
    // allowNo: true
  }),```